community/values/src/main/java/org/neo4j/values/storable/CoordinateReferenceSystem.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.values.storable;
+
+import org.neo4j.graphdb.spatial.CRS;
+
+public class CoordinateReferenceSystem implements CRS
```

Comment:
it really looks like this class should be enum instead?

Reply:
There are a few thousand CRS out there, normally defined by libraries. We expect to either:
* Include an appropriate dependency to populate this table
* Allow users to include such dependencies to extend this table for their domain

Right now we will not expose that capability, and we could have an enum, but it feels better to have the code sympathetic to the direction we expect it to go.

Reply:
ok sounds reasonable